configs/fischertechnik_TXT_initramfs_defconfig

```diff
@@ -0,0 +1,34 @@
+BR2_arm=y
+BR2_cortex_a8=y
+BR2_ARM_FPU_NEON=y
+BR2_ARM_INSTRUCTIONS_THUMB2=y
+BR2_GLOBAL_PATCH_DIR="$(BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH)/patches"
+BR2_PER_PACKAGE_DIRECTORIES=y
+BR2_TOOLCHAIN_EXTERNAL=y
+BR2_TOOLCHAIN_EXTERNAL_GDB_SERVER_COPY=y
+BR2_TARGET_OPTIMIZATION="-pipe"
+BR2_TARGET_GENERIC_HOSTNAME="FT-txt"
+BR2_TARGET_GENERIC_ISSUE="Welcome to FT-txt"
+BR2_INIT_NONE=y
+BR2_ROOTFS_DEVICE_CREATION_DYNAMIC_EUDEV=y
+BR2_ROOTFS_MERGED_USR=y
+# BR2_TARGET_ENABLE_ROOT_LOGIN is not set
+BR2_TARGET_GENERIC_GETTY_PORT="ttyO0"
+BR2_TARGET_GENERIC_GETTY_BAUDRATE_115200=y
+BR2_TARGET_GENERIC_GETTY_TERM="linux"
+BR2_ENABLE_LOCALE_WHITELIST="C en_US de de_AT de_DE de_CH fr es nl pt"
+BR2_GENERATE_LOCALE="en_US de_DE fr_FR nl_NL"
+BR2_ROOTFS_USERS_TABLES="$(BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH)/board/fischertechnik/TXT/user.tab"
+BR2_ROOTFS_OVERLAY="$(BR2_EXTERNAL_FTCOMMUNITY_TXT_PATH)/board/fischertechnik/TXT/initramfs"
+BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
+BR2_PACKAGE_E2FSPROGS=y
+BR2_PACKAGE_FBV=y
+BR2_PACKAGE_PSPLASH=y
+BR2_PACKAGE_LIBFUSE=y
+BR2_PACKAGE_LINUX_PAM=y
+# BR2_PACKAGE_IFUPDOWN_SCRIPTS is not set
+BR2_TARGET_ROOTFS_CPIO=y
+BR2_TARGET_ROOTFS_CPIO_GZIP=y
+# BR2_TARGET_ROOTFS_TAR is not set
+BR2_PACKAGE_HOST_KMOD=y
+BR2_PACKAGE_HOST_UBOOT_TOOLS=y
```

Comment:
mkfs.vfat is missing maybe?

Reply:
Maybe that is also why the mounting fails for you.

Reply:
no. Mounting fails due to permssion issues

Reply:
> mkfs.vfat is missing maybe?

I will check on that. But I'm quite sure it is already in place